src/dist/replication/meta_server/meta_bulk_load_service.h

```diff
@@ -175,6 +182,25 @@ class bulk_load_service
         return oss.str();
     }
 
+    inline bool is_partition_metadata_not_updated_unlock(gpid pid)
+    {
+        if (_partition_bulk_load_info.find(pid) != _partition_bulk_load_info.end()) {
```

Comment:
Better to save the find result temporary to avoid find it again when `_partition_bulk_load_info[pid]`